lib/Common/Memory/HeapBlock.cpp

```diff
@@ -930,6 +930,20 @@ void HeapBlock::PrintVerifyMarkFailure(Recycler* recycler, char* objectAddress,
                     return;
                 }
 
+                // Js::Type::entryPoint may contain outdated data uncleared, and reused by recycler
+                // Most often occurs with script function Type
+                if (offset ==
+#if TARGET_64
+                    0x18
```

Comment:
Can we not use the offsetof macro here instead of hardcoding the offset? type->entrypoint is generally safe to take a dependency on since its offset hasn't moved since Chakra started but it'd still be better to not hardcode this here. Also, I'd take a dependency on specific platforms rather than just x64 vs everything else since new compilers could do strange things 😄